src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/EnumExtentions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Exadel.CrazyPrice.Common.Models.Option;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Exadel.CrazyPrice.Common.Extentions
+{
+    public static class EnumExtentions
+    {
+        /// <summary>
+        /// Gets the formatted string from enum.
+        /// $k or $K - the key enum,
+        /// $v or $V - the value enum.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="formatString"></param>
+        /// <returns></returns>
```

Comment:
Check - seems ASP.NET will do this automatically - no need to worry. Just check in validator InEnum

Reply:
[Done](https://github.com/yegorl/exadel_discounts_be/pull/6/commits/92b05436440d8d29c426e3b8d7b0f62ea50d0784)